test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Indexers/FunctionIndexerTests.cs

```diff
@@ -316,76 +316,77 @@ public void GetSharedListenerIdOrNull_ReturnsExpectedValue()
-
-            // Act & Assert
-            await indexer.IndexMethodAsyncCore(method, indexCollector, CancellationToken.None);
-
-            Assert.Contains(loggerProvider.GetAllLogMessages(), x => x.FormattedMessage.Contains("Retries are not supported by the trigger binding for ") && x.Level == LogLevel.Warning);
-        }
-
-        [Fact]
-        public async Task IndexMethodAsyncCore_NoRetryAttribute_NoWarning()
-        {
-            // Arrange
-            var loggerFactory = new LoggerFactory();
-            var loggerProvider = new TestLoggerProvider();
-            loggerFactory.AddProvider(loggerProvider);
-
-            MethodInfo method = typeof(FunctionIndexerTests).GetMethod(nameof(FunctionIndexerTests.QueueTrigger_Test),
-                BindingFlags.Static | BindingFlags.Public);
-            Assert.NotNull(method); // Guard
-
-            FunctionIndexer indexer = FunctionIndexerFactory.Create(loggerFactory: loggerFactory);
-            var indexCollector = new TestIndexCollector();
-
-            // Act & Assert
-            await indexer.IndexMethodAsyncCore(method, indexCollector, CancellationToken.None);
-
-            Assert.DoesNotContain(loggerProvider.GetAllLogMessages(), x => x.FormattedMessage.Contains("Retries are not supported by the trigger binding for "));
-        }
-
-        [Theory]
-        [InlineData(typeof(FunctionIndexerTests), nameof(FunctionIndexerTests.QueueTriggerFixedDelayRetry_Test))]
-        [InlineData(typeof(QueueTriggerClassFixedDelayRetry), nameof(QueueTriggerClassFixedDelayRetry.QueueTriggerFixedDelayRetry_Test))]
-
-        public async Task IndexMethodAsyncCore_SupportsRetryAttribute_NoWarning(Type type, string methodName)
-        {
-            // Arrange
-            var loggerFactory = new LoggerFactory();
-            var loggerProvider = new TestLoggerProvider();
-            loggerFactory.AddProvider(loggerProvider);
-
-            TestTriggerBinsingProvider testTriggerBinsingProvider = new TestTriggerBinsingProvider();
-            MethodInfo method = type.GetMethod(methodName, BindingFlags.Static | BindingFlags.Public);
-            Assert.NotNull(method); // Guard
-
-            FunctionIndexer indexer = FunctionIndexerFactory.Create(loggerFactory: loggerFactory, triggerBindingProvider: testTriggerBinsingProvider);
-            var indexCollector = new TestIndexCollector();
-
-            // Act & Assert
-            // NotImplementedException is expected
-            await Assert.ThrowsAsync<NotImplementedException>(async () =>
-            {
-                await indexer.IndexMethodAsyncCore(method, indexCollector, CancellationToken.None);
-            });
-
-            Assert.DoesNotContain(loggerProvider.GetAllLogMessages(), x => x.FormattedMessage.Contains("Retries are not supported by the trigger binding for "));
-        }
+        // Temporary comment: https://github.com/Azure/azure-webjobs-sdk/issues/2788
+        //[Theory]
```

Comment:
Rather than comment out all the code, use the Skip property of Theory/Fact attributes to skip the tests. This way the code is still maintained, and we see skipped test warnings as reminders